launch/docker.go

```diff
@@ -45,11 +45,14 @@ func (d *docker) setupBin() error {
 
 	mount := fmt.Sprintf("%s:/opt/sd/", d.volume)
 	image := fmt.Sprintf("%s:%s", d.setupImage, d.setupImageVersion)
+	err = execCommand("docker", "pull", image).Run()
+	if err != nil {
+		return fmt.Errorf("failed to pull launcher image %v", err)
```

Comment:
[imo] Is it better to show docker image url to error message?

Reply:
We are now fixing the error output in this part.
So we'll fix it next PR, wait plz.

Reply:
I see. 👍